Tutorial/6_SimpleClientServer/SimpleClientServer.pproj

```diff
@@ -0,0 +1,10 @@
+<Project>
+<ProjectName>SimpleClientServer</ProjectName>
+<InputFiles>
+	<PFile>./PSrc/</PFile>
```

Comment:
Can we drop 6_SimpleClientServer from tutorial folder

Reply:
Done!